### PR TITLE
Add missing commas - Attempts to fix #485

### LINKED ISSUE #485: .awikstats module missing commas
This issue was automatically made by begiak, Apertium's beloved IRC bot, by the order of sushain on #apertium. A human is yet to update the description.

### PR DESCRIPTION
**Pull request by tomasff**
This attempts to fix #485.

I created a helper function `formatNumberThousands()`  formats the numbers with the thousands separator.
This is used in the `getStats()` function to format the numbers with the `,`.

### CODE REVIEW

File: modules/mailing_list.py [hunk 1]
```diff
@@ -68,10 +68,9 @@ def decode_mime_utf(text):
     return text
 
 def format_email(e, list_name):
-    #subject = e['Subject']
-    #subject = re.sub(r"(=\?.*\?=)(?!$)", r"\1 ", subject)
-
     subject = decode_mime_utf(e['Subject'])
+    subject = subject.replace('['+list_name.capitalize()+'] ', '')
```
Comment (Androbin) [inline modules/mailing_list.py]: Did you mean to include that in this PR?

Comment (Androbin) [reply modules/mailing_list.py]: It's because your previous PR was stashed, not merged. So you should rebase on master or else GitHub will make a scene.

Comment (tomasff) [reply modules/mailing_list.py]: I did not. I tried using different branches in my fork but it still added that change. If I close the branch where I commited that change, will it solve the issue?

Comment (tomasff) [reply modules/mailing_list.py]: @Androbin I'm not quite sure how to rebase on master

Comment (tomasff) [reply modules/mailing_list.py]: Update: I have rebased it and it should be fine now 👍

Comment (Androbin) [reply modules/mailing_list.py]:
```
git remote add upstream https://github.com/apertium/phenny.git
git fetch --all
git rebase upstream/master
```

Comment (tomasff) [reply modules/mailing_list.py]: Thank you. Is it fixed now? I removed the commit which was not necessary from this pull request using `git rebase`

**Comment by Androbin** [comment]
LGTM @jonorthwash

**Comment by jonorthwash** [comment]
Looks good to me too, but I'd like to see it tested before I merge.

**Comment by Androbin** [comment]
> I was unable to test it since I don't have access to the token.

@jonorthwash https://github.com/apertium/phenny/issues/485#issuecomment-564735327

**Comment by jonorthwash** [comment]
Okay, I checked out @tomasff's branch and tested.  Seems to not work, and gives me this info:
```
(23:33:18) firespeaker: .awikstats update sah
(23:33:34) begiak: INFO:root:Login as StemCounterBot succeeded
(23:33:35) begiak: Traceback (most recent call last):
(23:33:36) begiak:   File "/home/begiak/phenny/apertium_wikistats_bot.py", line 400, in <module>
(23:33:37) begiak:     fileCounts = getStats(jsonResponse, True)
(23:33:39) begiak:   File "/home/begiak/phenny/apertium_wikistats_bot.py", line 61, in getStats
(23:33:39) begiak:     statValue = formatNumberThousands(statValue)
(23:33:40) begiak:   File "/home/begiak/phenny/apertium_wikistats_bot.py", line 40, in formatNumberThousands
(23:33:41) begiak:     return '{:,}'.format(number)
(23:33:42) begiak: ValueError: Cannot specify ',' with 's'.
```

**Comment by tomasff** [comment]
@jonorthwash Should be fine now. I forgot to cast the statValue to an integer. I've also checked if the variable is an integer before trying to apply the `formatNumberThousands` funtion.

**Comment by Androbin** [comment]
Can `stat['value']` actually not be an integer?

**Comment by tomasff** [comment]
@Androbin I'm not sure. The initial problem was that the stat value was not cast to an int. I thought I'd check in case the API returned something which was not an integer. Should I remove the check?

**Comment by Androbin** [comment]
Unnecessarily defensive programming can be a bad habit.
Just check the docs: http://apertium.projectjj.com/stats-service/

**Comment by tomasff** [comment]
@Androbin right, thank you for the heads up. I'll remove it now.

**Comment by Androbin** [comment]
@jonorthwash The `Stats` schema doesn't seem to display the definition for `value` properly. Where is this model defined?

**Comment by Androbin** [comment]
Also, it's an OpenAPI, so why is `value` not sent directly as an `integer` directly?

**Comment by jonorthwash** [comment]
No idea.

**Comment by sushain97** [comment]
> @jonorthwash The Stats schema doesn't seem to display the definition for value properly. Where is this model defined?

https://github.com/apertium/apertium-stats-service/blob/master/src/models.rs#L87

Technically, the value is arbitrary.

> Also, it's an OpenAPI, so why is value not sent directly as an integer directly?

It is. e.g.

http://apertium.projectjj.com/stats-service/cat

**Comment by Androbin** [comment]
Fascinating. But does it ever hold something else than an integer or a string representing an integer?

**Comment by sushain97** [comment]
Well, it's supposed to be JSON. I don't think I have it returning anything but an integer right now.

If integers as strings are showing up, that must be something in the Python messing it up.

**Comment by sushain97** [comment]
Hmm, I couldn't find anything suspicious. Happy to debug further given an API request URL. There could be bad data in the DB from an old revision or something.

**Comment by Androbin** [comment]
Look no further than the link you just sent:
```
... "stat_kind":"Rules","value":"597"},
... "stat_kind":"Entries","value":24},
... "stat_kind":"Stems","value":95604},
... "stat_kind":"Paradigms","value":684},
```

**Comment by sushain97** [comment]
Ahh. I bet the issue is this missing a coercion to integer:

https://github.com/apertium/apertium-stats-service/blob/e7d645a27da3417f0d2673958c5e8c3b56f0425f/src/stats/mod.rs#L77

Opened an issue: https://github.com/apertium/apertium-stats-service/issues/62

**Comment by sushain97** [comment]
Should be fixed now. I migrated the old data and the link above now looks correct.

**Pull request merged**